src/__tests__/processHooks.platform.test.ts

```
@@ -0,0 +1,83 @@
+        disposed = true;
+      });
+
+      // Trigger the captured shutdown handler
+      return (adapter as any)._shutdown().then(() => {
+        expect(disposed).toBe(true);
+      });
+
+      expect(disposed).toBe(true);
+      // exit throws PlatformUnsupportedFunction but processHooks must swallow it
+      // If not swallowed, the test would fail.
+      unhook();
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)